doc/index.rst

```diff
@@ -112,14 +112,13 @@
 
    Work on SfePy is partially supported by the following ongoing projects:
 
+   - project GA23-06220S (Flexoelectric periodic structures for fluid transport
+     and energy harvesting) of the Czech Science Foundation, since 2023
+   - project LA 22-00863K (Controllable metamaterials and smart structures:
```

Comment:
This project ID is [GF22-00863K](https://www.isvavai.cz/cep?s=jednoduche-vyhledavani&ss=detail&n=0&h=GF22-00863K).

I would add links to the projects:

* GA23-06220S - the link is not yet available :)
* [GF22-00863K](https://www.isvavai.cz/cep?s=jednoduche-vyhledavani&ss=detail&n=0&h=GF22-00863K)
* [GA21-16406S](https://www.isvavai.cz/cep?s=jednoduche-vyhledavani&ss=detail&h=GA21-16406S)